homeassistant/components/openweathermap/config_flow.py

```diff
@@ -115,14 +116,13 @@
         )
 
     def _get_options_schema(self):
+        default_value = self.config_entry.options.get(CONF_MODE, DEFAULT_FORECAST_MODE)
+        if self.config_entry.options.get(CONF_MODE) == FORECAST_MODE_FREE_DAILY:
+            default_value = FORECAST_MODE_ONECALL_DAILY
```

Comment:
I just tested options form, if user have value `freedaily`, then select will be empty, and user have to choose option.
I propose to revert changes in this method and leave as it was.
Of course then coverage will be ok